foreman-selinux-enable

```diff
@@ -25,6 +29,10 @@ do
     /usr/sbin/semanage port -E | grep -q docker_port_t || \
       echo "port -a -t docker_port_t -p tcp 2375-2376" >> $TMP
 
+    if is_redhat_6; then
+      echo "port -a -t foreman_osapi_compute_port_t -p tcp 8774" >> $TMP
```

Comment:
Ok added, note I am not adding TMP_PORTS improvement in this patch, I will rebase the other one and add it here once we merge this one.